src/ui/TextLink.tsx

```diff
@@ -1,11 +1,11 @@
 'use client';
 
-import { Link, LinkProps, TextProps } from '@chakra-ui/react';
+import { Link, TextProps } from '@chakra-ui/react';
 import { forwardRef } from 'react';
 
 import { Text } from './Text';
 
-export type TextLinkProps = TextProps & LinkProps;
+export type TextLinkProps = TextProps & { href?: string; target?: string };
```

Comment:
Interesting that LinkProps didn't include these

Reply:
After update, text and link had conflicting props